robyn/argument_parser.py

```diff
@@ -38,20 +38,20 @@ def __init__(self):
         self.args = self.parser.parse_args()
 
     @property
-    def num_processes(self):
+    def num_processes(self) -> int:
         return self.args.processes
 
     @property
-    def workers(self):
+    def workers(self) -> int:
         return self.args.workers
 
     @property
-    def log_level(self):
+    def log_level(self) -> str:
         return self.args.log_level
 
     @property
-    def is_dev(self):
+    def is_dev(self) -> bool:
         _is_dev = self.args.dev
-        if _is_dev and (self.num_processes() != 1 or self.workers() != 1):
+        if _is_dev and (self.num_processes != 1 or self.workers != 1):
```

Comment:
Thank you for this suggestion. This is actually a code error and for some reason the code tests did not detect it.

Reply:
I chose to add types in the library as everyone else loves them.